vector/read.go

```diff
@@ -0,0 +1,432 @@
+package vector
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/netip"
+
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/pkg/nano"
+	"github.com/brimdata/zed/vng"
+	vngvector "github.com/brimdata/zed/vng/vector"
+	"github.com/brimdata/zed/zcode"
+
+	"github.com/RoaringBitmap/roaring"
```

Comment:
We just do two groups: standard library and everything else.
```suggestion
	"github.com/RoaringBitmap/roaring"
	"github.com/brimdata/zed"
	"github.com/brimdata/zed/pkg/nano"
	"github.com/brimdata/zed/vng"
	vngvector "github.com/brimdata/zed/vng/vector"
	"github.com/brimdata/zed/zcode"
```